mlflow/server/js/src/shared/web-shared/genai-traces-table/types.ts

```diff
@@ -168,7 +168,7 @@ export type TableFilter = {
   column: TracesTableColumnGroup | string;
   // Should be defined if a column group is used.
   key?: string;
-  operator: FilterOperator;
+  operator: FilterOperator | HiddenFilterOperator;
```

Comment:
missed this in backsync

```suggestion
  operator: FilterOperator;
```